plugins/grafana-custom-plugins/grafana-http-table-plugin/src/components/TablePanel.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useMemo } from 'react';
```

Comment:
Maybe we can add unit test for this file to verify if the data are in our component correctly?

Reply:
yes will do, working on getting tests to run locally